config/config.yaml

```diff
@@ -40,7 +40,7 @@ policy:
       # The build environment used for this repo. Right now, this value
       # is used for selecting the appropriate golang-cross image
       # used for building the binary, packages and docker images.
-      buildenv: 1.21-bullseye
+      buildenv: 1.22-bookworm
```

Comment:
aiui, this means dropping support for rhel8. Has this been decided?
Moreover, this change will apply to release-5.3 which needs to continue on 1.21-bullseye. 
Please also ensure that you adjust the plugin compiler to also follow these considerations.

Reply:
@andrei-tyk this should support rhel8, thanks @alephnull for spotting it